back-end/validation/register.js

```diff
@@ -1,6 +1,9 @@
 const Validator = require("validator");
 const isEmpty = require("is-empty");
 
+const userRegex = new RegExp(/^[a-zA-Z0-9]+([._]?[a-zA-Z0-9]+)*$/);
+const passRegex = new RegExp(/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{6,}$/);
```

Comment:
accidentally pushed password changes with username changes. 3rd commit has username changes only, this file in this commit can be ignored